demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx

```diff
@@ -116,6 +118,48 @@ export function ProductsGrid({ toolbarAction, rowAction }: Props): React.ReactEl
             flex: 1,
             minWidth: 150,
         },
+        {
+            field: "staticSelectType",
+            headerName: intl.formatMessage({ id: "product.staticSelectType", defaultMessage: "Type (static select)" }),
+            filterable: false,
+            sortable: false,
+            renderCell: ({ row }) => {
+                const primaryEmptyMessage = "-";
+                const typePrimaryLabels: Record<string, string> = {
+                    Cap: intl.formatMessage({ id: "product.staticSelectType.primaryText.Cap", defaultMessage: "This is a cap" }),
```

Comment:
I'd prefer FormattedMessage here, is that possible?

Reply:
Yes: bcbe408fe785d2215a3add2df5ed387d92a1164e